packages/beacon-node/test/perf/network/gossip/snappy.test.ts

```diff
@@ -0,0 +1,52 @@
+import {randomBytes} from "node:crypto";
+import * as snappyjs from "snappyjs";
+import * as snappy from "snappy";
+import {itBench} from "@dapplion/benchmark";
+
+describe("network / gossip / snappy", () => {
+  for (const msgLen of [100, 200, 300, 400, 500, 1000, 10000, 100000]) {
+    const uncompressed = randomBytes(msgLen);
+    const compressed = snappyjs.compress(uncompressed);
+    const RUNS_FACTOR = 1000;
+
+    itBench({
+      id: `snappyjs compress ${msgLen} bytes`,
+      runsFactor: RUNS_FACTOR,
+      fn: () => {
+        for (let i = 0; i < RUNS_FACTOR; i++) {
+          snappyjs.compress(uncompressed);
+        }
+      },
+    });
+
+    itBench({
+      id: `snappyjs uncompress ${msgLen} bytes`,
+      runsFactor: RUNS_FACTOR,
+      fn: () => {
+        for (let i = 0; i < RUNS_FACTOR; i++) {
+          snappyjs.uncompress(compressed);
+        }
+      },
+    });
+
+    itBench({
+      id: `snappy compress ${msgLen} bytes`,
+      runsFactor: RUNS_FACTOR,
+      fn: () => {
+        for (let i = 0; i < RUNS_FACTOR; i++) {
+          snappy.compressSync(uncompressed);
+        }
+      },
+    });
+
+    itBench({
+      id: `snappy uncompress ${msgLen} bytes`,
+      runsFactor: RUNS_FACTOR,
+      fn: () => {
+        for (let i = 0; i < RUNS_FACTOR; i++) {
+          snappy.uncompressSync(compressed);
+        }
+      },
+    });
```

Comment:
Tiny nit, can we add a describe block and change the perf order so the output is a bit easier to reason

```sh

 network / gossip / snappy
   compression
    ✔ snappyjs compress 100 bytes                                         371413.5 ops/s    2.692417 us/op        -        570 runs   2.04 s
    ✔ snappy compress 100 bytes                                           372290.0 ops/s    2.686078 us/op        -       1652 runs   4.97 s
    ✔ snappyjs compress 200 bytes                                         314647.7 ops/s    3.178158 us/op        -        451 runs   1.93 s
    ✔ snappy compress 200 bytes                                           413461.8 ops/s    2.418603 us/op        -        982 runs   2.88 s
    ✔ snappyjs compress 300 bytes                                         308590.0 ops/s    3.240546 us/op        -        506 runs   2.14 s
    ✔ snappy compress 300 bytes                                           370224.3 ops/s    2.701065 us/op        -       1742 runs   5.21 s
    ✔ snappyjs compress 400 bytes                                         277814.1 ops/s    3.599529 us/op        -        516 runs   2.36 s
    ✔ snappy compress 400 bytes                                           356533.9 ops/s    2.804782 us/op        -       1891 runs   5.81 s
    ✔ snappyjs compress 500 bytes                                         278411.4 ops/s    3.591807 us/op        -        716 runs   3.08 s
    ✔ snappy compress 500 bytes                                           399972.2 ops/s    2.500174 us/op        -       1435 runs   4.09 s
    ✔ snappyjs compress 1000 bytes                                        203750.4 ops/s    4.907966 us/op        -        316 runs   2.05 s
    ✔ snappy compress 1000 bytes                                          410775.5 ops/s    2.434420 us/op        -        545 runs   1.83 s
    ✔ snappyjs compress 10000 bytes                                       33580.64 ops/s    29.77906 us/op        -         53 runs   2.08 s
    ✔ snappy compress 10000 bytes                                         116049.7 ops/s    8.616998 us/op        -        161 runs   1.89 s
    ✔ snappyjs compress 100000 bytes                                      5677.085 ops/s    176.1467 us/op        -         12 runs   2.66 s
    ✔ snappy compress 100000 bytes                                        28890.04 ops/s    34.61400 us/op        -         25 runs   1.37 s

  decompression
    ✔ snappyjs uncompress 100 bytes                                       576763.1 ops/s    1.733814 us/op        -       1061 runs   2.36 s
    ✔ snappy uncompress 100 bytes                                         488977.0 ops/s    2.045086 us/op        -        946 runs   2.44 s
    ✔ snappyjs uncompress 200 bytes                                       450081.9 ops/s    2.221818 us/op        -       1422 runs   3.66 s
    ✔ snappy uncompress 200 bytes                                         473192.9 ops/s    2.113303 us/op        -       1781 runs   4.26 s
    ✔ snappyjs uncompress 300 bytes                                       411732.2 ops/s    2.428763 us/op        -       1101 runs   3.18 s
    ✔ snappy uncompress 300 bytes                                         449624.3 ops/s    2.224079 us/op        -       1286 runs   3.37 s
    ✔ snappyjs uncompress 400 bytes                                       343024.8 ops/s    2.915241 us/op        -        733 runs   2.64 s
    ✔ snappy uncompress 400 bytes                                         413890.0 ops/s    2.416101 us/op        -       1827 runs   4.92 s
    ✔ snappyjs uncompress 500 bytes                                       500098.0 ops/s    1.999608 us/op        -       1343 runs   3.19 s
    ✔ snappy uncompress 500 bytes                                         379704.5 ops/s    2.633627 us/op        -       2106 runs   6.05 s
    ✔ snappyjs uncompress 1000 bytes                                      316446.9 ops/s    3.160088 us/op        -        163 runs   1.02 s
    ✔ snappy uncompress 1000 bytes                                        441280.0 ops/s    2.266135 us/op        -        861 runs   2.45 s
    ✔ snappyjs uncompress 10000 bytes                                     40791.23 ops/s    24.51507 us/op        -         48 runs   1.69 s
    ✔ snappy uncompress 10000 bytes                                       156750.8 ops/s    6.379553 us/op        -        787 runs   5.52 s
    ✔ snappyjs uncompress 100000 bytes                                    4033.521 ops/s    247.9223 us/op        -         12 runs   3.71 s
    ✔ snappy uncompress 100000 bytes                                      32567.37 ops/s    30.70558 us/op 
```

Reply:
:+1: details matter